custodia/cli/__init__.py

```diff
@@ -187,12 +187,22 @@ def handle_name_value(args):
 
 def handle_plugins(args):
     result = []
+    errmsg = "**ERR** {0} ({1.__class__.__name__}: {1})"
     for plugin in PLUGINS:
         result.append('[{}]'.format(plugin))
         eps = pkg_resources.iter_entry_points(plugin)
         eps = sorted(eps, key=operator.attrgetter('name'))
         for ep in eps:
-            result.append(str(ep))
+            try:
+                if hasattr(ep, 'resolve'):
+                    ep.resolve()
+                else:
+                    ep.load(require=False)
+            except Exception as e:  # pylint: disable=broad-except
+                if args.verbose:
+                    result.append(errmsg.format(ep, e))
+            else:
```

Comment:
It's correct: try/except/else. Without verbose or debug, it shows only available plugins.